### ISSUE TITLE
Add an accessible keyboard shortcuts panel

### ISSUE THREAD

**Issue by Ovgodd** [suitenumerique/docs#1830]
### Is your feature request related to a problem or unsupported use case? Please describe.

Currently, several keyboard shortcuts are available in the application  but they are not explicitly discoverable by users.

While these shortcuts follow common patterns, their behavior may not be obvious to all users, especially:
- keyboard-only users,
- users relying on assistive technologies,
- users unfamiliar with the application's interaction model.

This can lead to reduced discoverability, learning friction, and confusion about available keyboard interactions.

@rl-83 we can discuss about it together !

---

### Describe the solution you'd like

Add an **accessible list of keyboard shortcuts** within the application.

The list would:
- document existing shortcuts
- briefly explain their behavior,
- be accessible via keyboard and screen readers.

The list does not need to be visible by default but should be easy to discover.

Possible entry points:
- a “Keyboard shortcuts” panel or dialog,
- an entry in a Help menu,
- a shortcut such as `ctrl + shift + /`.

### Do you want to work on it through a Pull Request?

Of course yes !

**Comment by robin-lecomte** [comment]
Hello Cyril,

I have two suggestions that could be valuable to extend to the entire LaSuite:  
1. Include a keyboard shortcuts sheet in the app’s documentation. It’s the easiest option, but it’s less seamlessly integrated since you have to leave the app and check the documentation.
2. Add a keyboard shortcuts modal accessible through the new support module (see attached image) and a dedicated keyboard shortcut.

What do you think?

<img width="1906" height="634" alt="Image" src="https://github.com/user-attachments/assets/4d0d51e9-fc95-4ac1-9d8f-0b6c7bbea14f" />

**Comment by Ovgodd** [comment]
> Hello Cyril,
> 
> I have two suggestions that could be valuable to extend to the entire LaSuite:
> 
> 1. Include a keyboard shortcuts sheet in the app’s documentation. It’s the easiest option, but it’s less seamlessly integrated since you have to leave the app and check the documentation.
> 2. Add a keyboard shortcuts modal accessible through the new support module (see attached image) and a dedicated keyboard shortcut.
> 
> What do you think?
> 
> <img alt="Image" width="1906" height="634" src="https://private-user-images.githubusercontent.com/185491142/544415344-4d0d51e9-fc95-4ac1-9d8f-0b6c7bbea14f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzAyMTY1MTYsIm5iZiI6MTc3MDIxNjIxNiwicGF0aCI6Ii8xODU0OTExNDIvNTQ0NDE1MzQ0LTRkMGQ1MWU5LWZjOTUtNGFjMS05ZDhmLTBiNmM3YmJlYTE0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDIwNFQxNDQzMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZmE0NjA0NGNkYTc5ZTQ3Yjk1MGJmOWI0ODNhOWQzNGU4Y2Q2NGQzOTAzNGI5NGQ5MzE5YTkxZTZmZWU5NzQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._3xvw0g9ri0wkcowneGPXTnSapA3tBeztvF4geUprVc">

Hey @rl-83 this is exactly what I haved in mind ! ( the second choice ) I like it :) !!

**Comment by Ovgodd** [comment]
@cyberbaloo has created a doc listing the available shortcuts in Docs. First draft available here: https://docs.numerique.gouv.fr/docs/5dad64c8-42de-4638-8a87-f265bf50bcf2/
